apps/condo/domains/ticket/utils/handlers.js

```diff
@@ -113,12 +136,86 @@ const handleTicketEvents = async (requestData) => {
     }
 
     if (eventTypes[STATUS_CHANGED_EVENT_TYPE]) {
-        // TODO(DOMA-2434): Add logic for sending notifications to resident on ticket status change
+        let type
```

Comment:
why just type? ticket status type? message type?

Reply:
it should be ticket status type